### ISSUE TITLE
add MSE loss as another loss function

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#3]
See: https://pytorch.org/docs/stable/nn.html#mseloss, you'll also want to modify the function `get_label_coding` inside `data_loader.py` to properly encode the negative categories using this loss (the negative examples should have the value '0' using this loss)

**Comment by alexatartaglini** [comment]
Added! Should reduction should be 'none,' 'sum,' or 'mean?'

**Comment by wkvong** [comment]
Let's stick with the default reduction which is 'mean', and see how it goes for now!
